tm2py/components/demand/prepare_demand.py

```diff
@@ -195,11 +195,16 @@ def _read_demand(self, file_config, time_period, num_zones):
     def prepare_household_demand(self):
         """Prepares highway and transit household demand matrices from trip lists produced by CT-RAMP.
         """
+        iteration = self.controller.iteration
         
+        # Create folders if they don't exist
```

Comment:
@xyzjayne Maybe better to error out than creating the folder? I'm thinking what if there is a typo in the file path the user specified.

Reply:
This might still be useful for someone who doesn't know the file structure trying to run the model only with some warmstart input files. It's not overwriting any file and might prevent some headache.